core/integration/features/transfers/one_off_transfers.feature

```diff
@@ -40,11 +44,64 @@ Scenario: simple successful transfers (0057-TRAN-001, 0057-TRAN-007, 0057-TRAN-0
     Then "576380694832d9271682e86fffbbcebc09ca79c259baa5d4d0298e12ecdee303" should have general account balance of "20000" for asset "VEGA"
     Then "a3c024b4e23230c89884a54a813b1ecb4cb0f827a38641c66eeca466da6b2ddf" should have general account balance of "30000" for asset "VEGA"
 
+# Scenario: discount is applied for transaction fee (0057-TRAN-013)
```

Comment:
Curious why this scenario can't be salvaged

Reply:
Yeah, it will need to be re-worked by @Jiajia-Cui in another PR